x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -117,7 +118,7 @@ export const AddExceptionModal = memo(function AddExceptionModal({
     Array<ExceptionListItemSchema | CreateExceptionListItemSchema>
   >([]);
   const [fetchOrCreateListError, setFetchOrCreateListError] = useState<ErrorInfo | null>(null);
-  const { addError, addSuccess } = useAppToasts();
+  const { addError, addSuccess, addWarning } = useAppToasts();
```

Comment:
Not in any way blocking this PR, but just thinking, I think this component would benefit from using `useReducer`. The number of `useState`s are growing and (just personally) working on adding error states in here recently, felt like it was getting crowded.

Reply:
Makes sense to me! I don't think I will address it here, but if I get time, will take a stab at that soon.